src/main/java/com/minecolonies/coremod/util/InventoryUtils.java

```diff
@@ -1342,17 +1348,19 @@ public static int getFirstOpenSlotFromProviderForSide(@NotNull final ICapability
      * @param facing       The side to check for.
      * @param toolTypeName The toolTypeName of the tool to find.
      * @param minimalLevel The minimal level to find.
+     * @param maximumLevel The maximum level to find.
      * @return True if a Tool with the given toolTypeName was found in the given
      * {@link ICapabilityProvider}, false when not.
      */
-    public static boolean isToolInProviderForSide(@NotNull final ICapabilityProvider provider, @Nullable EnumFacing facing, @NotNull final String toolTypeName, int minimalLevel)
+    public static boolean isToolInProviderForSide(@NotNull final ICapabilityProvider provider, @Nullable EnumFacing facing, @NotNull final String toolTypeName, final int minimalLevel,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)